Project.toml

```diff
@@ -0,0 +1,14 @@
+name = "OrderedCollections"
+uuid = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
+version = "2.0.0-DEV"
```

Comment:
```suggestion
version = "1.2.0"
```

Reply:
✔️  Assuming we also add the suggestion of deprecate,